docs/windows/README.md

```diff
@@ -0,0 +1,473 @@
+
+**IPFS repository cleanup:**
+
+```cmd
+cd "C:\Program Files\PinShare"
+
+# Run IPFS garbage collection
+ipfs.exe --repo-dir="C:\ProgramData\PinShare\ipfs" repo gc
+```
+
+**Limit IPFS resource usage:**
+
+1. Edit: `C:\ProgramData\PinShare\ipfs\config`
+2. Modify `Swarm.ConnMgr`:
+   ```json
+   "ConnMgr": {
+     "HighWater": 300,
+     "LowWater": 150
+   }
+   ```
+
+### Can't Connect to Peers
+
+1. **Check firewall** - Ensure ports 4001 and 50001 are open
+2. **Check NAT** - PinShare uses relay for NAT traversal
+3. **View peer status**:
+   ```cmd
+   curl http://localhost:9090/api/status
+   ```
+
+### Logs and Debugging
+
+**View logs:**
+
+```cmd
+# Service log
+type "C:\ProgramData\PinShare\logs\service.log"
+
+# IPFS log
+type "C:\ProgramData\PinShare\logs\ipfs.log"
+
+# PinShare log
+type "C:\ProgramData\PinShare\logs\pinshare.log"
+```
+
+**Enable debug mode:**
+
+1. Stop the service
+2. Run in console mode:
+   ```cmd
+   cd "C:\Program Files\PinShare"
+   pinsharesvc.exe debug
+   ```
+3. Watch console output
+
+**Tail logs in PowerShell:**
+
+```powershell
+Get-Content "C:\ProgramData\PinShare\logs\service.log" -Wait -Tail 50
+```
+
+## Uninstallation
+
+### Using Control Panel
+
+1. Open Settings → Apps → Installed apps
+2. Find "PinShare"
+3. Click "Uninstall"
+
+### Using Installer
+
+```cmd
+msiexec /x PinShare-Setup.msi
+```
+
+### Manual Cleanup (if needed)
+
+The uninstaller preserves data. To completely remove:
+
+```cmd
+# Remove program files
+rmdir /s "C:\Program Files\PinShare"
+
+# Remove data (WARNING: Deletes all pins and configuration)
+rmdir /s "C:\ProgramData\PinShare"
+
+# Remove registry entries
+reg delete "HKLM\SOFTWARE\PinShare" /f
+reg delete "HKCU\SOFTWARE\PinShare" /f
+```
+
+## Advanced Topics
+
+### Running Multiple Instances
+
+To run multiple PinShare instances:
+
+1. Install normally (first instance)
+2. For additional instances:
+   - Copy installation directory
+   - Change all ports in configuration
+   - Install as separate service with different name
+
+### Backup and Restore
+
+**Backup:**
+
+```cmd
+# Stop service
+net stop PinShareService
+
+# Backup data directory
+xcopy "C:\ProgramData\PinShare" "D:\Backup\PinShare\" /E /I /H
+
+# Restart service
+net start PinShareService
+```
+
+**Restore:**
+
+```cmd
+# Stop service
+net stop PinShareService
+
+# Restore data
+xcopy "D:\Backup\PinShare\" "C:\ProgramData\PinShare\" /E /I /H /Y
+
+# Restart service
+net start PinShareService
+```
+
+### Performance Tuning
+
+**For archive nodes** (storing many files):
+- Increase disk space for IPFS repo
+- Disable automatic garbage collection
+- Add to config: `"archive_node": true`
+
+**For low-resource systems:**
+- Reduce IPFS connection limits
+- Disable caching: `"enable_cache": false`
+- Enable VirusTotal skip: `"skip_virus_total": true`
+
+### Network Configuration
+
+**Static IP / Public Access:**
+
+If you want to access PinShare from other computers:
+
+1. **Change bind address** (advanced):
```

Comment:
Are we sure that this is possible given the current interface? I commented on this elsewhere - let's make sure we reconcile this section with whatever we end up doing regarding ensuring support for interface configuration.